code/controllers/configuration/entries/jobs.dm

```diff
@@ -0,0 +1,184 @@
+// This file pretty much just handles all of the interactions between jobconfig.toml and the codebase. This is started by work originating in SSconfig, so I'm okay with it being here.
+
+/// Initializes all of the config singletons for each job config type and adds it to the `job_config_datum_singletons` var list.
+/datum/controller/subsystem/job/proc/generate_config_singletons()
```

Comment:
s a bit odd to have subsystem procs outside of the subsystem's file, what was the reason for that? just the interaction between this and configs?

Reply:
yes, SSjob was like a thousand lines long of just miscellaneous stuff, felt like it had to do something.